measures/2019/measures-schema.yaml

```diff
@@ -301,6 +301,7 @@ definitions:
       - geriatrics
       - urgentCare
       - skilledNursingFacility
+      - generalPracticeFamilyMedicine
```

Comment:
For clarification, This type was added as cahps measures use `generalPracticeFamilyMedicine` and the schema was being invalidated because of this. Please let me know if this change is incorrect.

Reply:
Hi, Sam. If there were no errors generated when you ran the build script, this change looks fine to me.

Reply:
Okay! I ran `npm test` and did a npm build for 2019 measures and they passed. I'll go ahead and merge.